lib/tasks/us_census_api.rake

```diff
@@ -0,0 +1,96 @@
+namespace :us_census_api do
+  desc "Uses the census API to pull down all the variables we need into a usable form and then do some post-processing if needed"
+  task :all, [] => [:shapes, :vars, :import, :coc_agg, :test]
```

Comment:
With a census api key and after rake db:seed runs to completion, this should work.